scripts/Muon/GUI/Common/plot_widget/model_fit_pane/plot_model_fit_pane_presenter.py

```diff
@@ -0,0 +1,47 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+from mantidqt.utils.observer_pattern import GenericObserverWithArgPassing
+from Muon.GUI.Common.plot_widget.fit_pane.plot_fit_pane_presenter import PlotFitPanePresenter
+
+# A 10% margin when updating the x range based on the data x limits
+X_MARGIN = 0.1
+
+
+class PlotModelFitPanePresenter(PlotFitPanePresenter):
+
+    def __init__(self, view, model, context, fitting_context, figure_presenter):
+        super().__init__(view, model, context, fitting_context, figure_presenter)
+        self._data_type = [""]
+        self._sort_by = [""]
+        self.update_view()
+
+        self.update_override_tick_labels_observer = GenericObserverWithArgPassing(
+            self.update_override_x_and_y_tick_labels)
+        self.update_x_range_observer = GenericObserverWithArgPassing(self.update_x_plot_range)
+
+        self._figure_presenter.set_autoscale(True)
+        self._figure_presenter.set_errors(True)
+        self._figure_presenter.set_plot_as_point_data(True)
```

Comment:
I think the line style and data points should come from the context, either we pass them as plot arguments or the figure_presenter gets them direct from the relevant plotting context.